fix: added examples to apex run and apex get (#505)
salesforcecli/plugin-apex: messages/runtest.md

  <%= config.bin %> <%= command.id %> --test-level RunLocalTests --output-dir <path to outputdir> --target-org me@my.org

- Run all tests in the org asynchronously:
  
  <%= config.bin %> <%= command.id %> --target-org myscratch

> - Run all tests synchronously. The command waits to display the test results until all tests finish:

Reviewer: ```suggestion
- Run all tests synchronously; the command waits to display the test results until all tests finish:
```

  
  <%= config.bin %> <%= command.id %> --synchronous

> - Run specific tests using the --test-level flag:

Reviewer 2: The original in the DX Developer Guide has
> Use --test-level to run a subset of tests. See [Understanding Deploy](https://developer.salesforce.com/docs/atlas.en-us.250.0.apexcode.meta/apexcode/apex_deploying_ant_deploy.htm) in the Apex Developer Guide for the list of possible test level values.

Can we still keep that link given that it contains useful information?
![Screenshot 2024-06-27 at 10 26 53 AM](https://github.com/salesforcecli/plugin-apex/assets/139700604/adfbce5d-bd7d-4855-84be-b15f14b82930)

Author: Thanks for your feedback @daphne-sfdc . I have a question about one of the NoTestRun values. We don't see it in the source -- https://github.com/salesforcecli/plugin-apex/blob/main/src/commands/apex/run/test.ts#L26. Can you confirm that it's a valid value? Thanks!

Reviewer 2: Copying over my response from Slack:

> I reread the docs and it looks like NoTestRun shouldn't be a valid value for a command used to run tests, since you would be asking the command to run nothing.  It looks like it should only be used for whether tests should be run during deployment, and the default is deploying does not run tests.  So you're right that there should only be 3 options for the `--test-level` flag.

  <%= config.bin %> <%= command.id %> --test-level RunLocalTests

> - Run Apex tests on all the methods a the specified class; output results in Test Anything Protocol (TAP) format, and requests code coverage result:

Reviewer: ```suggestion
- Run Apex tests on all the methods in the specified class; output results in Test Anything Protocol (TAP) format and request code coverage results:
```

> - Run Apex tests on all the methods in the specified class; output results in Test Anything Protocol (TAP) format and request code coverage results:

Reviewer 2: It looks like this doc in the CLI Command Reference does not have any "References" or "See Also" section.  Would it be possible to add one and move the reference to TAP here?  (The example that uses TAP is now in the CLI Command Reference instead of the DX Developer Guide but the link to the TAP reference is still in the DX Developer Guide.)

TAP website: https://testanything.org/

Existing reference in DX Developer Guide:
![Screenshot 2024-06-27 at 10 35 17 AM](https://github.com/salesforcecli/plugin-apex/assets/139700604/a24a437b-c988-4fcf-8268-7bfdfcc81a5a)

Author: I'll confirm but I don't think the command reference has a See Also section.

  <%= config.bin %> <%= command.id %> --class-names TestA --class-names TestB --result-format tap --code-coverage

- Run Apex tests on methods specified using the standard Class.method notation; if you specify a test class without a method, the command runs all methods in the class:
  
  <%= config.bin %> <%= command.id %> --tests TestA.excitingMethod --tests TestA.boringMethod --tests TestB

> - Run Apex tests on methods specified using the standard Class.method notation with a namespace; if you specify a test class without a method, the command runs all methods in the class:

Reviewer: ```suggestion
- Run Apex tests on methods specified using the standard Class.method notation with a namespace:
```

Reviewer: No need to repeat all the previous example blurb.
